Documentation/toolbox.md

```diff
@@ -71,12 +82,12 @@ kubectl create -f toolbox.yaml
 
 Wait for the toolbox pod to download its container and get to the `running` state:
 ```bash
-kubectl -n rook-ceph get pod rook-ceph-tools
+kubectl -n rook get pod -l "app=rook-tools"
 ```
 
 Once the rook-ceph-tools pod is running, you can connect to it with:
 ```bash
-kubectl -n rook-ceph exec -it rook-ceph-tools bash
+kubectl -n rook exec -it $(kubectl -n rook get pod -l "app=rook-tools" -o jsonpath='{.items[0].metadata.name}') bash
```

Comment:
i'm just wishing the `exec` command could be simpler, but it doesn't appear that there is a better way than this.

Reply:
the documentation has a couple other places that should be updated with this command as well, for example:
- at the end of this doc where the deployment should be deleted instead of a pod
- upgrade.md
- common-issues.md

There are also the integration tests, but for simplicity i think let's keep the pod there and not worry about changing it to a deployment.